src/praisonai-agents/test_conversation_simple.py

```
@@ -0,0 +1,121 @@
+    
+    # Test conversation compaction
+    compactor = IntelligentConversationCompactor(analyzer)
+    
+    # Create longer conversation for compaction
+    long_messages = []
+    for i in range(10):
+        long_messages.append({"role": "user", "content": f"This is a longer user message {i} with enough content to exceed token limits and trigger compaction"})
+        long_messages.append({"role": "assistant", "content": f"This is a longer assistant response {i} providing detailed information and guidance"})
+    
+    # Test compaction
+    result, compaction_context = compactor.compact_conversation(
```

Ruff (0.15.15)

[warning] Unpacked variable `compaction_context` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)